spec/commands/install_spec.rb

```diff
@@ -321,6 +321,28 @@
       expect(File.exist?(bundled_app("Gemfile.lock"))).to eq(true)
     end
 
+    it "throws an error if a gem is added twice in Gemfile without version requirements" do
+      install_gemfile <<-G
+        source "file://#{gem_repo2}"
+        gem "rack"
+        gem "rack"
+      G
+
+      expect(out).to include("Your Gemfile lists the gem rack (>= 0) more than once.")
+      expect(out).to include("You should probably keep only one of them")
+    end
+
+    it "throws an error if a gem is added twice in Gemfile with version requirements" do
+      install_gemfile <<-G
+        source "file://#{gem_repo2}"
+        gem "rack", "1.0"
+        gem "rack", "1.1"
+      G
+
+      expect(out).to include("You cannot specify the same gem twice with different version requirements")
+      expect(out).to include("If you want to update the gem version, run `bundle update rack`. You may need to change the version requirement specified in the Gemfile if it's too restrictive")
```

Comment:
This should have the same message as the one above. No mention of bundle update.

Reply:
I am still unable to figure out how to show different messages based on commands `add` and `install`